frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -27,24 +28,18 @@ export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses,
   );
   const numberOfIssues = Object.values(metrics).reduce((acc, cur) => acc + cur, 0);
   const hasIssues = riskEntries.length > 0 && numberOfIssues > 0;
-  const isInit = isInitialized(metrics);
+
+  const isWaitingOrDisabled = _isWaitingOrDisabled(metrics);
+  const isError = _isError(metrics);
 
   return (
     <div className="co-insights__box">
-      {!isInit && (
-        <div className="co-status-popup__section">
-          Insights identifies and prioritizes risks to security, performance, availability, and
-          stability of your clusters.
-        </div>
-      )}
-      {isInit && (
-        <div className="co-status-popup__section">
-          Your cluster has been identified, and Insights analyzes your cluster. The results will be
-          displayed shortly.
-        </div>
+      {isError && <div className="co-status-popup__section">Temporary unavailable.</div>}
+      {isWaitingOrDisabled && (
+        <div className="co-status-popup__section">Disabled or waiting for results.</div>
```

Comment:
This is better, but it would be nice if there was a way to differentiate these two states.

Reply:
@spadgett, unfortunately, there is no easy way how to deal with it right now. We are planning to do a spike and rethink the whole state logic for the widget. But for now, we need to get rid of the incorrect state in the scope of this bug.